### PR TITLE
Update histograms.md

### PR DESCRIPTION
**Pull request by SylwiaOliwia2**
Added `Accessing the y-axis values` section

### CODE REVIEW

File: doc/python/histograms.md [hunk 1]
```diff
@@ -69,6 +69,32 @@ fig = px.histogram(df, x="total_bill", nbins=20)
 fig.show()
 ```
 
+#### Accessing the y-axis values
```
Comment (emmanuelle) [inline doc/python/histograms.md]: Maybe `Accessing the counts (y-axis) values`?

**Comment by emmanuelle** [comment]
Thank you for the example @SylwiaOliwia2 ! A lot of people get confused about this, it will be very useful. I'm just curious to know why you used pandas functions instead of `np.histogram`, is it because some people working with pandas prefer to use only pandas functions? I post below a version using `np.histogram` so that we can compare and discuss about the two solutions.
```
import plotly.express as px
import numpy as np
df = px.data.tips()
# create the bins
counts, bins = np.histogram(df.total_bill, bins=range(0, 60, 5))
bins = 0.5 * (bins[:-1] + bins[1:])
fig = px.bar(x=bins, y=counts, labels={'x':'total_bill', 'y':'count'})
fig.show()
```

**Comment by SylwiaOliwia2** [comment]
Indeed, I used to pandas, this solution was the most obvious solution to me.
However, `numpy.histogram` is much simpler and using continuous x-scale! If you agree I'll keep only the NumPy solution.

**Comment by emmanuelle** [comment]
@SylwiaOliwia2 I agree that I find the numpy solution simpler.

**Comment by SylwiaOliwia2** [comment]
@emmanuelle done :)

**Comment by emmanuelle** [comment]
Thank you @SylwiaOliwia2 ! Now that you have contributed several doc PRs to plotly.py, do you have any feedback about the contribution process? Any way that we could improve it? Do you feel like trying to work on a bug or a part of the code for your next PR ;-) ? Great to have here anyway!

**Comment by SylwiaOliwia2** [comment]
> Thank you @SylwiaOliwia2 ! Now that you have contributed several doc PRs to plotly.py, do you have any feedback about the contribution process? Any way that we could improve it? Do you feel like trying to work on a bug or a part of the code for your next PR ;-) ? Great to have here anyway!

Thank you, @emmanuelle :) Definitely I can try working out on the code the next time. Feel free to suggest any issue :)

Regarding contribution, using tags is very helpful (that's how I've found the issue), especially for newbies. So keep using them, please ;)

**Pull request merged**